translations/fr/try_ruby_10.md

```diff
@@ -0,0 +1,26 @@
+---
+lang:   EN
+title:  Vous avez 30 minutes? Essayez Ruby!
+answer:
+class:  stretcher chapmark
+ok:
+error:
+---
+
+Ruby est un language de programmation originaire du Japon et qui a
+revolutionné le développement logiciel.
+
+La beauté de Ruby se situe dans sa balance entre simplicité et puissance.
+
+Vous pouvez saisir du code Ruby dans l'éditeur et utiliser les boutons suivants pour naviguer:
+
+- __Run__ &rarr; Execute le code dans l'éditeur
+- __Copy__ &rarr; Copie l'example de code dans l'éditeur
+- __Next__ &rarr; Vous permet de sauter à la leçon suivante
+- __Back__ &rarr; Vous permet de revenir à la leçon précédente
+- __Clear__ &rarr; Remets l'éditeur à zéro
+
+Utilise le bouton 'planête' <span aria-hidden="true" class="glyphicon glyphicon-globe"></span>
```

Comment:
Perhaps we should use double-quotes here? E.g.:

- `« planête »`
- `❝planête❞`
- `"planête"`

Reply:
First one's the most correct for [French "guillemets"](https://en.wikipedia.org/wiki/Guillemet). Yet.